### PR TITLE
Update FAQ.md

### LINKED ISSUE #4685: How to configure scopes for own Web APIs?
### Core Library

MSAL.js v2 (@azure/msal-browser)

### Wrapper Library

MSAL Angular (@azure/msal-angular)

### Documentation Location

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/msal-interceptor.md

### Description

Hello Documentation team

We are using MSAL authentication library for our Angular 13 Project. I am trying to understand how we can use MSAL Interceptor for passing tokens as per the documentation. We are not able to understand "how to configure scopes for custom APIs?"

As per my little understanding, we need to configure scope on Azure Portal where we generated the client id. 
Could you please arrange to clarify?

### PR DESCRIPTION
**Pull request by ellymakuba**

### CODE REVIEW

File: lib/msal-browser/FAQ.md [hunk 1]
```diff
@@ -406,3 +406,6 @@ MSAL.js will only process tokens which it originally requested. If your flow req
 ## What should I do if I believe my issue is with the B2C service itself rather than with the library
 
 In that case, please file a support ticket with the B2C team by following the instructions here: [B2C support options](https://docs.microsoft.com/azure/active-directory-b2c/support-options).
+
+## Register custom scopes for a web API
+[How to register custom scopes for my web API](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-configure-app-expose-web-apis).
```
Comment (jasonnutter) [inline lib/msal-browser/FAQ.md]: Can you please move this up underneath the "Scopes & Resources" heading?

Also, please remove `en-us` from the link:

```suggestion
[How to register custom scopes for my web API](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis).
```

**Review by gustavoabel — APPROVED** [review]

File: lib/msal-browser/FAQ.md (hunk 1 above)
Comment (jasonnutter) [reply lib/msal-browser/FAQ.md]: In addition to moving this under the "scopes and resources" section, can you add a link to teh the "scopes and resources" heading on this page from this section in the MSAL Angular docs: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/ellymakuba-patch-1/lib/msal-angular/docs/v2-docs/msal-interceptor.md#protected-resource-map

**Review by gustavoabel — APPROVED** [review]

**Review by jasonnutter — CHANGES_REQUESTED** [review]
Please address above feedback, thanks!

**Review by bmahall — APPROVED** [review]
Kindly merge in response to #4685 post addressing the feedback.  Thanks! @ellymakuba

File: lib/msal-browser/FAQ.md (hunk 1 above)
Comment (ellymakuba) [reply lib/msal-browser/FAQ.md]: I made these changes.

File: lib/msal-browser/FAQ.md [hunk 2]
```diff
@@ -328,12 +328,15 @@ Token lifetimes are 1 hour and the session lifetime is 24 hours. This means that
 
 Please see the documentation on [Tenancy in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/develop/single-and-multi-tenant-apps#who-can-sign-in-to-your-app)
 
-# Scopes & Resources
+# Scopes & Resources - [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/ellymakuba-patch-1/lib/msal-angular/docs/v2-docs/msal-interceptor.md#protected-resource-map)
```
Comment (jasonnutter) [inline lib/msal-browser/FAQ.md]: Sorry, I mean link to this FAQ from the msal-interceptor.md page.

Comment (ellymakuba) [reply lib/msal-browser/FAQ.md]: I have addressed this comment, please confirm it's resolved.

**Review by jasonnutter — APPROVED** [review]

**Pull request merged**